skimage/viewer/__init__.py

```diff
@@ -1,6 +1,6 @@
-from .._shared.utils import warn
 from .viewers import ImageViewer, CollectionViewer
 from .qt import has_qt
 
 if not has_qt:
-    warn('Viewer requires Qt')
+    from warnings import warn
+    warn('Viewer requires Qt', stacklevel=2)
```

Comment:
Should be 1?

Reply:
Yea, could be 1. I tested this quite a bit, doesn't actually matter on what you select.